### PR TITLE
[platforms] update BeagleBone Black default configuration

### LINKED ISSUE #1144: Update BeagleBone Black default configuration
The BeagleBone Black default configuration currently builds using Network Manager aligning with an old configuration. This should be updated to match the other platforms.

Align the default beaglebone configurations with the raspbian configuration. Both are debian based Linux systems with the same packages available. The setup documentation should be aligned with this change to avoid removing connman in favor of network-manager.

### PR DESCRIPTION
**Pull request by srickardti**
fixes #1144

### CODE REVIEW

File: script/_disable_services [hunk 1]
```diff
@@ -36,9 +36,14 @@ bbb_disable_services()
     SERVICE_LIST=""
     SERVICE_LIST="${SERVICE_LIST} apache2 " # Debian jessie
     SERVICE_LIST="${SERVICE_LIST} nginx "   # Debian stretch
+    SERVICE_LIST="${SERVICE_LIST} bonescript-autorun.service"
+    SERVICE_LIST="${SERVICE_LIST} bonescript-autorun.service"
```
Comment (wgtdkp) [inline script/_disable_services:40]: Are the two lines duplicated?

File: examples/platforms/beagleboneblack/default [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 #
-#  Copyright (c) 2017, The OpenThread Authors.
+#  Copyright (c) 2021, The OpenThread Authors.
```
Comment (wgtdkp) [inline examples/platforms/beagleboneblack/default:3]: @jwhui Should this be the year when this file is created?

**Review by wgtdkp — APPROVED** [review]

File: examples/platforms/beagleboneblack/default (hunk 2 above)
Comment (jwhui) [reply examples/platforms/beagleboneblack/default:3]: We should retain the original year, but can indicate the update as follows:
```suggestion
#  Copyright (c) 2017-2021, The OpenThread Authors.
```

**Review by jwhui — APPROVED** [review]
Thanks! 👍🏼

**Pull request merged**